src/dbnode/client/session.go

```diff
@@ -488,6 +495,9 @@ func (s *session) recordWriteMetrics(consistencyResultErr error, respErrs int32,
 	}
 	if consistencyResultErr == nil {
 		s.metrics.writeSuccess.Inc(1)
+		if state.successAsLeavingAndInitializingCountTowardsConsistency {
+			s.metrics.writeSuccessForCountLeavingAndInitializingAsPair.Inc(1)
```

Comment:
isn't this going to double count? 

- afaict both `writeSuccess` and `writeSuccessForCountLeavingAndInitializingAsPair` have metric name `write.success`.

Reply:
Thanks for pointing.
Fixed